sw/device/silicon_creator/rom/data/rom_testplan.hjson

```diff
@@ -134,400 +121,1008 @@
-            - Sign the ROM_EXT with a valid key.
-            - Test runner loads the ROM and the ROM_EXT.
-            - Attempt to boot.
-            - Verify that the chip indicates the correct boot fault.
+            `OWNER_SW_CFG_ROM_BOOTSTRAP_EN` OTP item must be `kHardenedBoolTrue` (`0x739`).
+
+            - Apply bootstrap pin strapping.
+            - Reset the chip.
+            - Verify that the chip responds to `READ_STATUS` (`0x05`) with `0x00`.
+
+            See `rom_e2e_bootstrap_enabled_requested`.
+            '''
+      tags: ["rom", "verilator", "dv", "fpga", "silicon"]
+      milestone: V2
+      tests: []
+    }
+
+    {
+      name: rom_e2e_bootstrap_watchdog_disabled
+      desc: '''Verify that watchdog is disabled upon entering bootstrap.
+
+            `OWNER_SW_CFG_ROM_BOOTSTRAP_EN` OTP item must be `kHardenedBoolTrue` (`0x739`).
+            `OWNER_SW_CFG_ROM_WATCHDOG_BITE_THRESHOLD_CYCLES` OTP item must be `0x30d40`
+
+            - For TEST, DEV, PROD, PROD_END, and RMA life cycle states:
+              - Apply bootstrap pin strapping.
+              - Reset the chip.
+              - Verify that the chip responds to `READ_STATUS` (`0x05`) with `0x00`.
+              - Release bootstrap pin strapping and wait for 2 seconds.
+                - Note: Watchdog is always disabled in TEST and RMA. In other states, the threshold
+                  is set to `OWNER_SW_CFG_ROM_WATCHDOG_BITE_THRESHOLD_CYCLES`.
+              - Verify that the chip responds to `READ_STATUS` (`0x05`) with `0x00`.
+              - Verify that there was no output from UART.
+            '''
+      tags: ["rom", "verilator", "dv", "fpga", "silicon"]
+      milestone: V2
+      tests: []
+    }
+
+    {
+      name: rom_e2e_bootstrap_read_id
+      desc: '''Verify the JEDEC ID used during bootstrap.
+
+            `OWNER_SW_CFG_ROM_BOOTSTRAP_EN` OTP item must be `kHardenedBoolTrue` (`0x739`).
+
+            - Apply bootstrap pin strapping.
+            - Reset the chip.
+            - Verify that the chip responds to `READ_JEDEC_ID` (`0x9f`) with
+              - 12 repetitions of the continuation code `0x7f`,
+              - manufacturer ID `0xef`, and
+              - density `0x14`.
+            '''
+      tags: ["rom", "verilator", "dv", "fpga", "silicon"]
+      milestone: V2
+      tests: []
+    }
+
+    {
+      name: rom_e2e_bootstrap_read_sfdp
+      desc: '''Verify the SFDP table used during bootstrap.
+
+            `OWNER_SW_CFG_ROM_BOOTSTRAP_EN` OTP item must be `kHardenedBoolTrue` (`0x739`).
+
+            - Apply bootstrap pin strapping.
+            - Reset the chip.
+            - Verify that the chip responds to `READ_SFDP` (`0x5a`).
+            - Verify the SFDP header structure. See this
+              [document](https://docs.google.com/document/d/1X_x7f62IrPeHRLGBzybhDu9TsvsSCb6_1uKOjQO_fI4/edit?resourcekey=0-vLsVqyrt1F2mGwLrz73uwA#heading=h.gc2hf662pvy9).
```

Comment:
We should move these docs to markdown. Ok if it is under a bootstrap section under ROM docs.

Reply:
You're right, created #14443. Thanks!